### PR TITLE
Update Codepen examples to https

### PR DESCRIPTION
**Pull request by raisedadead**
Refers #7700 
To Close #7825 in favor of this PR (Solved the merge issues in the earlier PR by @theflametrooper)

This PR addresses the following challenges:

| Challenge | Fixed |
| --- | --- |
| Build a Tribute Page | Yes |
| Build a Personal Portfolio Webpage |  |
| Build a Random Quote Machine | Yes |
| Show the Local Weather |  |
| Build a Wikipedia Viewer | Yes |
| Use the Twitchtv JSON API | Yes |
| Build a JavaScript Calculator | Yes |
| Build a Pomodoro Clock | Yes |
| Build a Tic Tac Toe Game | Yes |
| Build a Simon Game |  |
| Build a Markdown Previewer | Yes |
| Build a Camper Leaderboard | Yes |
| Build a Recipe Box | Yes |
| Build the Game of Life | Yes |
| Build a Roguelike Dungeon Crawler Game | Yes |
| Visualize Data with a Bar Chart |  |
| Visualize Data with a Scatterplot Graph | Yes |
| Visualize Data with a Heat Map | Yes |
| Show Relationships with a Force Directed Graph |  |
| Map Data Across the Globe |  |

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-ziplines.json [hunk 1]
```diff
@@ -202,9 +202,9 @@
       "challengeType": 3,
       "titleRu": "Создайте сайт-портфолио",
       "descriptionRu": [
-        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a> которое функционально соответствует вот этому: <a href='http://codepen.io/FreeCodeCamp/full/VemmoX/' target='_blank'>http://codepen.io/FreeCodeCamp/full/VemmoX/</a>.",
+        "<strong>Задание:</strong> Создайте приложение <a href='https://codepen.io' target='_blank'>CodePen.io</a> которое функционально соответствует вот этому: <a href='https://codepen.io/FreeCodeCamp/full/VemmoX/' target='_blank'>https://codepen.io/FreeCodeCamp/full/VemmoX/</a>.",
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/basic-ziplines.json]: Shouldn't the link here be changed to https://codepen.io/hallaathrad/full/vNEPpL? I used Google Translate on the surrounding text and it appears this is the personal portfolio page.

File: seed/challenges/01-front-end-development-certification/basic-ziplines.json [hunk 2]
```diff
@@ -218,9 +218,9 @@
       ],
       "titleEs": "Construye una página web para tu portafolio",
       "descriptionEs": [
-        "<strong>Objetivo:</strong> Crea una aplicación con <a href='http://codepen.io' target='_blank'>CodePen.io</a> cuya funcionalidad sea similar a la de esta: <a href='http://codepen.io/FreeCodeCamp/full/VemmoX/' target='_blank'>http://codepen.io/FreeCodeCamp/full/VemmoX/</a>.",
+        "<strong>Objetivo:</strong> Crea una aplicación con <a href='https://codepen.io' target='_blank'>CodePen.io</a> cuya funcionalidad sea similar a la de esta: <a href='https://codepen.io/FreeCodeCamp/full/VemmoX/' target='_blank'>https://codepen.io/FreeCodeCamp/full/VemmoX/</a>.",
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/basic-ziplines.json]: Change to https://codepen.io/hallaathrad/full/vNEPpL as well? Uses the same `/VemmoX/` link

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 3]
```diff
@@ -61,15 +61,15 @@
...
         "<strong>User Story:</strong> I can toggle between sorting the list by how many brownie points they've earned in the past 30 days and by how many brownie points they've earned total.",
-        "<strong>Hint:</strong> To get the top 100 campers for the last 30 days: <a href='http://fcctop100.herokuapp.com/api/fccusers/top/recent' target='_blank'>http://fcctop100.herokuapp.com/api/fccusers/top/recent</a>.",
-        "<strong>Hint:</strong> To get the top 100 campers of all time: <a href='http://fcctop100.herokuapp.com/api/fccusers/top/alltime' target='_blank'>http://fcctop100.herokuapp.com/api/fccusers/top/alltime</a>.",
+        "<strong>Hint:</strong> To get the top 100 campers for the last 30 days: <a href='https://fcctop100.herokuapp.com/api/fccusers/top/recent' target='_blank'>http://fcctop100.herokuapp.com/api/fccusers/top/recent</a>.",
+        "<strong>Hint:</strong> To get the top 100 campers of all time: <a href='https://fcctop100.herokuapp.com/api/fccusers/top/alltime' target='_blank'>http://fcctop100.herokuapp.com/api/fccusers/top/alltime</a>.",
```
Comment (erictleung) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: This doesn't really affect functionality but the text for the link isn't `https` but to stay consistent, shouldn't these be changed to `https` as well?

**Comment by erictleung** [comment]
@raisedadead Looks good otherwise! 👍

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 3 above)
Comment (raisedadead) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: Agreed.

File: seed/challenges/01-front-end-development-certification/basic-ziplines.json (hunk 2 above)
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/basic-ziplines.json]: I'll fork @hallaathrad 's pen under FCC and update this. Thanks for catching.

**Comment by erictleung** [comment]
@raisedadead LGTM! :+1:

**Pull request merged**